src/lib/autoinstall/entries/importer.rb

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "autoinstall/entries/registry"
+
+module Y2Autoinstallation
+  module Entries
+    # Worker class that handles importing of profile section using info from {Description}.
+    # Its ability is beside calling import on auto client also detecting unhandled
+    # or obsolete sections.
+    class Importer
```

Comment:
It is nice to have a service class that implements the importing process. However, I am not sure whether it should live in the `Entries` module. Perhaps I would consider having it under `Y2Autoinstallation` (`src/lib/autoinstall`).

Reply:
it is fine for me to move it. BTW when I am at it, I am thinking about having instead of client_path some object that represent calling side of auto clients to encapsulate it, but probably not in this PR as it can grow too much.